arrow/arrow.py

```diff
@@ -767,12 +788,13 @@ def time(self):
         return self._datetime.time()
 
     def timetz(self):
-        ''' Returns a ``time`` object with the same hour, minute, second, microsecond and tzinfo. '''
+        ''' Returns a ``time`` object with the same hour, minute, second, microsecond and
+        tzinfo. '''
 
         return self._datetime.timetz()
 
     def astimezone(self, tz):
-        ''' Returns a ``datetime`` object, adjusted to the specified tzinfo.
+        ''' Returns a ``datetime`` object, converted to the specified timezone.
```

Comment:
I tried to keep the same words used in index.rst: replace vs convert.  So I changed "adjusted" here for consistency.